source/intel/asm/crc32c_sse42_asm.c

```diff
@@ -14,15 +14,20 @@
 #        pragma clang diagnostic ignored "-Wdollar-in-identifier-extension"
 #    endif
 
+#ifdef __APPLE__
```

Comment:
is this really an `__APPLE__` thing? or is it a`__CLANG__` thing?

I'm wary of this from years of seeing `#ifdef _WIN32` that should be `#ifdef _MSC_VER`

we probably can't `#ifdef` on whether deadstripping is happening, since it's a linker flag, not compiler flag, right?

or can we do something to mark this file/function/asm-block "no deadstripping here please"?

Reply:
It's an Apple linker thing -- at least, it doesn't exist in GNU ld and so doesn't exist on my AL2 version of Clang.